modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -13,10 +13,26 @@ import * as Options from './Options';
 import { createUploader, openNotification } from './util/ImageUploader';
 
 /**
+ * TinyMCE Editor Upload API
  * Handles image uploads, updates undo stack and patches over various internal functions.
  *
- * @private
  * @class tinymce.EditorUpload
+ * @example
+ * // Apply a filter to all images as uploaded..
+ * tinymce.activeEditor.EditorUpload.addFilter((image) => {
+ *   const maxSize = 1920 * 1080;
+ *   const imageSize = image.width * image.height;
+ *   return imageSize < maxSize;
+ * });
+ *
+ * // UploadImages
+ * tinymce.activeEditor.EditorUpload.uploadImages();
+ *
+ * // ScanForImages
+ * tinymce.activeEditor.EditorUpload.scanForImages();
+ *
+ * // Destroy
```

Comment:
Destroy is not meant to be called by developers it's just an internal thing that just happens to be exposed. So maybe not add an example of that.

Reply:
Done, thanks